Jellyfin.Api/Helpers/DynamicHlsHelper.cs

```diff
@@ -539,12 +683,29 @@ private int GetBitrateVariation(int bitrate)
             return variation;
         }
 
-        private string ReplaceBitrate(string url, int oldValue, int newValue)
+        private string ReplaceVideoBitrate(string url, int oldValue, int newValue)
         {
             return url.Replace(
                 "videobitrate=" + oldValue.ToString(CultureInfo.InvariantCulture),
                 "videobitrate=" + newValue.ToString(CultureInfo.InvariantCulture),
                 StringComparison.OrdinalIgnoreCase);
         }
+
+        private string ReplaceProfile(string url, string codec, string oldValue, string newValue)
+        {
+            return url.Replace(
+                codec + "-profile=" + oldValue,
+                codec + "-profile=" + newValue,
+                StringComparison.OrdinalIgnoreCase);
```

Comment:
Slight optimization.

```suggestion
            string profileStr = codec + "-profile=";
            return url.Replace(
                profileStr + oldValue,
                profileStr + newValue,
                StringComparison.OrdinalIgnoreCase);
```